changehc/README.md

```diff
@@ -0,0 +1,72 @@
+# Change Healthcare Indicator
+
+COVID-19 indicator using outpatient visits from Change Healthcare claims data.
+Reads claims data into pandas dataframe.
+Makes appropriate date shifts, adjusts for backfilling, and smooths estimates.
+Writes results to csvs.
+
+
+## Running the Indicator
+
+The indicator is run by directly executing the Python module contained in this
+directory. The safest way to do this is to create a virtual environment,
+installed the common DELPHI tools, and then install the module and its
+dependencies. To do this, run the following code from this directory:
+
+```
+python -m venv env
+source env/bin/activate
+pip install ../_delphi_utils_python/.
+pip install .
+```
+
+*Note*: you may need to install blas, in Ubuntu do
+```
+sudo apt-get install libatlas-base-dev gfortran
+```
+
+All of the user-changable parameters are stored in `params.json`. To execute
```

Comment:
is this file currently present? I only see the template

Reply:
My bad, I needed to force add it

Reply:
It's deliberate that `params.json` is gitignored, since it often contains parameters that change daily (like dates or filenames) and things like API keys.